include/alpaka/test/event/EventHostManualTrigger.hpp

```diff
@@ -686,13 +701,7 @@ namespace alpaka::trait
             //   on host updates may hang. This includes synchronization between the host and
             //   the device build upon value-based HIP queue synchronization APIs such as
             //   cuStreamWaitValue32() and cuStreamWriteValue32().
-#    if BOOST_COMP_NVCC
-            ALPAKA_UNIFORM_CUDA_HIP_RT_CHECK(hipCUResultTohipError(cuStreamWaitValue32(
-                static_cast<CUstream>(queue.getNativeHandle()),
-                reinterpret_cast<CUdeviceptr>(event.m_spEventImpl->m_devMem),
-                0x01010101u,
-                CU_STREAM_WAIT_VALUE_GEQ)));
-#    else
```

Comment:
We don't support the path alpaka -> HIP -> CUDA anymore so this can be removed entirely.